pelican/generators.py

```diff
@@ -727,8 +727,13 @@ def generate_output(self, writer):
             source_path = os.path.join(self.path, sc.source_path)
             save_as = os.path.join(self.output_path, sc.save_as)
             mkdir_p(os.path.dirname(save_as))
-            shutil.copy2(source_path, save_as)
             logger.info('Copying %s to %s', sc.source_path, sc.save_as)
+            try:
+                # workaround for Android python copy2 bug ([issue28141])
+                shutil.copy2(source_path, save_as)
+            except OSError as e:
+                logger.warning("A problem occurred copying file %s to %s; %s",
+                             source_path, save_as, e)
```

Comment:
`flake8` is complaining here because this line is ["under-indented"](https://travis-ci.org/getpelican/pelican/jobs/161134330#L186). IIRC, it wants the indent to line up with the opening paranthesis on the previous line (ie. add 2 spaces to the indentation).